.hound.yml

```diff
@@ -2,7 +2,11 @@ ruby:
   config_file: ./ruby/.rubocop.yml
 
 jshint:
-  config_file: ./javascript/.jshint
+  enabled: false
```

Comment:
Why this change?

Reply:
I didn't end up running a test. I think it's ok to have jshint explicitly disabled just to be on the safe side...

Reply:
Ok.